internal/forms/forms.go

```diff
@@ -732,6 +776,9 @@ func (m *Manager) fillFormTemplate(absPathTemplate string, formDestURL string, p
 	nowDateUTC := now.UTC().Format("2006-01-02Z")
 	nowTimeUTC := now.UTC().Format("15:04:05Z")
 	udtg := strings.ToUpper(now.UTC().Format("021504Z Jan 2006"))
+	nowGPS := gpsPos(false, false)
```

Comment:
I would suggest separating the tasks of getting the position from the GPS device vs formatting the position into different methods. Since we're having to do some costly communication with the GPS device, doing that three times seems like overkill.